tests/php/src/MobileRedirectionTest.php

```diff
@@ -151,7 +156,7 @@ public function test_redirect_on_transitional_and_not_available() {
 		AMP_Options_Manager::update_option( Option::THEME_SUPPORT, AMP_Theme_Support::TRANSITIONAL_MODE_SLUG );
 		AMP_Options_Manager::update_option( Option::ALL_TEMPLATES_SUPPORTED, false );
 		AMP_Options_Manager::update_option( Option::SUPPORTED_TEMPLATES, [ 'is_author' ] );
-		$this->go_to( amp_add_paired_endpoint( '/' ) );
```

Comment:
Going to the AMP endpoint seems to have been a bug here. I'm unclear why it was passing tests before.